{"name": "test_wrong_pwd", "status": "passed", "description": "密码输入6次错误", "attachments": [{"name": "log", "source": "ff2fdae6-532e-4572-8d59-faf0f2865e6b-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "4ff1041f-22ae-48ac-b638-cd9c77b5aa15-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "afbed76d-5ebd-403f-ae5a-e9236fe42048-attachment.txt", "type": "text/plain"}], "start": 1631502409999, "stop": 1631502411353, "uuid": "49058c10-4e5c-4f67-be1c-5fe1e760a03e", "historyId": "7b9480ebb1961c7c14cb69cc7e65ba50", "testCaseId": "e4c5762512a76c681290f066bcc0e861", "fullName": "test_case.Login_case.TestLogin#test_wrong_pwd", "labels": [{"name": "story", "value": "参数填写不正确，登录失败"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Login_case"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "9220-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Login_case"}]}